src/test/addrman_tests.cpp

```diff
@@ -697,16 +697,17 @@ BOOST_AUTO_TEST_CASE(addrman_serialization)
     auto addrman_asmap1_dup = std::make_unique<AddrMan>(netgroupman, DETERMINISTIC, ratio);
     auto addrman_noasmap = std::make_unique<AddrMan>(EMPTY_NETGROUPMAN, DETERMINISTIC, ratio);
 
-    CDataStream stream(SER_NETWORK, PROTOCOL_VERSION);
+    DataStream stream{};
+    const CAddress::SerParams ser_params{{CNetAddr::Encoding::V1}, CAddress::Format::Network};
```

Comment:
You construct `V1` params a couple of times in net_processing as well, just thought having them already done might be easier? You'd be saying `WithParams(CAddress::V1_DISK, info)` rather than `WithParams(ser_params, info)` or whatever, so it'd be slightly clearer inline as well as maybe saving a redeclaration?